### ISSUE TITLE
running the tests

### ISSUE THREAD

**Issue by melnimr** [dimikout3/MarsExplorer#9]
When trying to run the tests:

python3 mars-explorer/tests/test.py

 I get

gym.error.Error: No registered env with id: exploConf-v01

**Comment by dimikout3** [comment]
Hello, it seems that the environment is not installed in the gym. Check for errors earlier, during the installment process. Let me know if you find anything.

**Comment by melnimr** [comment]
To be honest I don't know what you mean by installation, are we talking about the pip install -e thing?

**Comment by dimikout3** [comment]
Yes, you must somehow let the "open-ai gym" now that you have created a new environment called "exploConf-v01". Check if it is registered in your open-ai gym environments

**Comment by mmyyen** [comment]
I find sometime if you change the ```exploConf-v01 ``` into ```exploConf-v1 ```. it might work
